### ISSUE TITLE
Get_First_Transaction.py gives date of last transaction.

### ISSUE THREAD

**Issue by 0tokens** [RandstormBTC/randstorm#9]
Get_First_Transaction.py gives date of last transaction. 
I appreciate your ongoing effort very much!

**Comment by brahmastra** [comment]
yes, line 19:  first_transaction = transactions[0] this is last transaction
change  to  first_transaction = transactions[len(transactions)-1]

**Comment by 0tokens** [comment]
Hi Brahmastra, thanks for providing a solution! Although it seems like this project is abandoned.

**Comment by ghost** [comment]
> Hi Brahmastra, thanks for providing a solution! Although it seems like this project is abandoned.

A few of us still discuss this project on telegram so feel free to join, but hopefully Karl returns soon
t.me/randstorm

**Comment by brahmastra** [comment]
Problem is what btcscan org  API show only 25 last  tx,  better use blockchain info
